base/Dockerfile

```diff
@@ -2,11 +2,21 @@
 FROM gcr.io/google_appengine/debian8
 
 # Install updates and dependencies
-RUN apt-get update -y && apt-get install --no-install-recommends -y -q curl python build-essential git ca-certificates libkrb5-dev imagemagick && \
+RUN apt-get update -y && apt-get install --no-install-recommends -y -q curl python build-essential git ca-certificates libkrb5-dev imagemagick wget && \
     apt-get clean && rm /var/lib/apt/lists/*_*
 
+# Add the files necessary for verifying and installing node
+# These files are placed in /opt/nodejs so that they will remain
+# even if the /nodejs directory is removed.  Further, the PATH is
+# is not updated, because these scripts should only be called
+# using an absolute path.
+RUN mkdir /opt/nodejs
+ADD KEYS /opt/nodejs
+ADD verify_node /opt/nodejs
+ADD verify_install_node /opt/nodejs
```

Comment:
I think this can be shortened to `ADD KEYS verify_node verify_install_node /opt/nodejs/`.